cdpctl/validation/infra/issue_templates.yml

```diff
@@ -13,86 +13,110 @@ docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-creat
 id: AWS_ROLE_REQUIRES_ACTIONS_FOR_ALL_S3_RESOURCES
 summary: "The role ({0}) requires the following actions for all S3 resources"
 render_type: list
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-iam-policy-definitions.html
 ---
 id: AWS_ROLE_FOR_DATA_BUCKET_MISSING_ACTIONS
 summary: "The role ({0}) requires the following actions for the S3 data location ({1})."
 render_type: list
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-iam-policy-definitions.html#autoId4
 ---
 id: AWS_IDBROKER_ROLE_REQUIRES_ACTIONS_FOR_ALL_RESOURCES
 summary: "The role ({0}) requires the following actions for resource wildcard (*)."
 render_type: list
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-iam-policy-definitions.html#autoId0
 ---
 id: AWS_ROLE_REQUIRES_ACTIONS_FOR_ALL_EC2_RESOURCES
 summary: "The role ({0}) requires the following actions for all EC2 resources ([*])"
 render_type: list
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-iam-permissions.html
 ---
 id: AWS_ROLE_REQUIRES_ACTIONS_FOR_ALL_RESOURCES
 summary: "The role ({0}) requires the following actions for all resources ([*])"
 render_type: list
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-iam-permissions.html
 ---
 id: AWS_ROLE_REQUIRES_ACTIONS_FOR_SERVICE_ROLL_RESOURCES
 summary: "The role ({0}) requires the following actions for all resources (\"arn:aws:iam::*:role/aws-service-role/*\")"
 render_type: list
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-iam-permissions.html
 ---
 id: AWS_ROLE_REQUIRES_ACTIONS_FOR_LOG_PATH
 summary: "The role ({0}) requires the following actions for the log storage path ({1}):"
 render_type: list
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-iam-policy-definitions.html#autoId1
 ---
 id: AWS_ROLE_REQUIRES_ACTIONS_FOR_LOG_BUCKET
 summary: "The role ({0}) requires the following actions for the log storage bucket ({1}):"
 render_type: list
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-iam-policy-definitions.html#autoId1
 ---
 id: AWS_LOGGER_INSTANCE_PROFILE_SHOULD_CONTAIN_LOGGER_ROLE
 summary: "The logger instance profile {0} set in config {1} should contain a logger role."
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-idbroker-minimum-setup.html#autoId1
 ---
 id: AWS_LOGGER_ROLE_SHOULD_HAVE_EC2_TRUST
 summary: "The logger role {0} should contain a trust policy for EC2"
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-idbroker-minimum-setup.html#autoId1
 ---
 id: AWS_S3_BUCKET_INVALID
 summary: "The s3a url {0} is invalid."
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-aws-req-s3.html
 ---
 id: AWS_S3_BUCKET_DOES_NOT_EXIST
 summary: "S3 bucket {0} does not exist."
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-aws-req-s3.html
```

Comment:
We could also go without a link for this one as it's self-explanatory. Probably also for forbidden access below. I'm fine either way.

Reply:
Yeah thought it it a bit but at least it gives them a link to say ... "oh yeah its it is in the docs"